pages/pipelines/hosted_agents/code_access.md

```diff
@@ -18,3 +19,48 @@ To use a private GitHub repository with Buildkite hosted agents, you'll need to
 ## Public repositories
 
 Buildkite does not require any special permissions to access public repositories.
+
+## Other private repositories
+
+There are 2 key aspects to utilizing a private repository with Buildkite hosted agents.
+
+1. Adding an SSH key as a Secret to the Buildkite hosted agent cluster.
+1. Adding the plugin to the initial pipeline steps, and any further steps within the pipeline that are loaded.
```

Comment:
> ... within the pipeline that are loaded.

That are loaded to where? (The verb "load" is transitive and so it's missing the object here.)
I'm assuming that you mean the pipeline being uploaded into Buildkite to start building? If so...

```suggestion
1. Add the plugin to the initial pipeline steps, and any further steps within the uploaded pipeline.
```